test/e2e/e2e_test.go

```diff
@@ -57,6 +57,36 @@ var _ = Describe("controller", Ordered, func() {
 		outputCm, err := cmdCm.CombinedOutput()
 		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Failed to deploy cert manager: %s", outputCm))
 
+		By("validating the cert-manager-webhook pod is running as expected")
```

Comment:
Thanks, Test `should run successfully` has a similar code block, should we make similar pieces into a function and reference it in both the tests?

Reply:
Yes, that's a good idea. There is redundancy there.

Reply:
Done